test/generate.test.ts

```diff
@@ -93,7 +93,7 @@ describe('generateClassAssignment ', () => {
         'declare const prop5: any;',
         'declare const property: my_assembly.Property;',
         '',
-        'const classA = new my_assembly.ClassA(this, \'MyClassA\', {',
+        'const classA = new my_assembly.ClassA(\'scope\', \'MyClassA\', {',
```

Comment:
this has to happen. i could not find a good way of mocking the `constructs.Construct` fqn and spent far too much time trying to figure it out.